src/sentry/static/sentry/app/views/organizationRateLimits/index.jsx

```diff
@@ -119,25 +119,29 @@ const RateLimitEditor = React.createClass({
 
     return (
       <form onSubmit={this.onSubmit}>
-        <p>Your organization is limited to <strong>{maxRate} events per minute</strong>.
-          When this rate is exceeded the system will begin discarding data until the
-          next interval.</p>
+          <p>
+            {tct('Your organization is limited to [strong:[maxRate] events per minute]. When this rate is exceeded the system will begin discarding data until the next interval.',
+              {
+                strong: <strong/>,
+                maxRate: maxRate
```

Comment:
we should leave a comment there somewhere that this might not translate well to all languages because of maxRate being a variable that might affect the plural form of "events per minute".  We do not have a solution within tct for this yet.